rewards/boost/calc_boost.py

```diff
@@ -51,12 +52,13 @@ def badger_boost(current_block: int, chain: str) -> Dict[str, Any]:
         calc_stake_ratio(addr, native_setts, non_native_setts) for addr in all_addresses
     ]
     stake_ratios = dict(zip(all_addresses, stake_ratios_list))
-
+    nfts = fetch_nfts(chain, current_block)
     for addr in all_addresses:
```

Comment:
we should probably refactor this into an init function.  Makes it more readable for the main function and allows us to create more focused testing